episodes/3-documentation.md

```diff
@@ -346,6 +346,107 @@ MkDocs and Sphinx:
 - create nicely-formatted HTML pages out of .md or .rST files
 - programming language independent
 
+#### MkDocs
+
+::: challenge
+### Exercise: Generate MkDocs documentation
+
+Before we start generating documentation, make sure that you have installed mkdocs:
+
+```bash
+conda list mkdocs
```

Comment:
That's to check if mkdocs is available, but there are no instruction to install it. Maybe add a conda and/or pip command needed to install it ?